src/main/java/com/epam/esm/entity/SqlQueries.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.esm.entity;
```

Comment:
This class should be located in some other package. It's not an entity

Reply:
Created a package called sql and moved it there